Source/Core/Core/HW/DSPLLE/DSPDebugInterface.cpp

```diff
@@ -116,7 +116,7 @@ void DSPDebugInterface::ToggleBreakpoint(unsigned int address)
   }
 }
 
-bool DSPDebugInterface::IsMemCheck(unsigned int address)
+bool DSPDebugInterface::IsMemCheck(unsigned int address, u32 size)
```

Comment:
unsigned int vs u32?
It is an alias, but I'd prefer to have the same in the declaration and implementation.